message_center_compassion/view/gmc_message_view.xml

```diff
@@ -29,6 +29,8 @@
             <field name="arch" type="xml">
                 <tree string="Message Pool" colors="blue:state == 'new';black:state == 'pending';gray:state == 'success';red:state == 'failure'" create="false">
                     <field name="action_id" />
+                    <field name="partner_id" />
+                    <field name="child_id" />

```

Comment:
Remove this field from the view, because module `message_center` does not depend on `child_compassion`. You should add it only in module `child_compassion`